python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -435,31 +468,15 @@ def launch_run(self, context: LaunchRunContext) -> None:
             del run_task_kwargs["launchType"]
 
         # Run a task using the same network configuration as this processes's task.
-        response = self.ecs.run_task(**run_task_kwargs)
-
-        tasks = response["tasks"]
-
-        if not tasks:
-            failures = response["failures"]
-            failure_messages = []
-            for failure in failures:
-                arn = failure.get("arn")
-                reason = failure.get("reason")
-                detail = failure.get("detail")
-
-                failure_message = (
-                    "Task"
-                    + (f" {arn}" if arn else "")
-                    + " failed."
-                    + (f" Failure reason: {reason}" if reason else "")
-                    + (f" Failure details: {detail}" if detail else "")
-                )
-                failure_messages.append(failure_message)
-
-            raise Exception("\n".join(failure_messages) if failure_messages else "Task failed.")
+        task = backoff(
+            self._run_task,
+            retry_on=(RetryableEcsException,),
+            kwargs=run_task_kwargs,
+            max_retries=5,
```

Comment:
Should this maybe be configurable, even if via env var? If it defaulted to 0 it would preserve the existing behavior right? Just wondering if a user running in a single AZ would want this to be on by default

Should the backoff maybe be configurable as well? That might be more relevant for the single AZ case

Reply:
sure - i can add some configurability in